_observing-your-data/ad/dashboards-anomaly-detection.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Integrating anomaly detection with Dashboard
+parent: Anomaly detection
+nav_order: 50
+---
+
+# Integrating anomaly detection with Dashboard
+Introduced 2.9
+{: .label .label-purple }
+
+Anomaly detection provides an automated means of detecting harmful outliers and protects your data. When you enable anomaly detection for a metric, OpenSearch applies algorithms to continuously analyze systems and applications, determine normal baselines, and surface anomalies. 
+
+You can connect data visualizations to OpenSearch datasets and then create, run, and view anomaly alarms and results from visualizations in the **Dashboard** interface. With only a couple clicks you can bring together traces, metrics, and logs to make your applications and infrastructure fully observable.
+
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.9 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## General requirements for anomaly detection visualizations
+
+Anomaly detection visualizations are intended for use with time-series data (real-time or historical). The following visualization types are used frequently for time-series data: line, area, vertical bar, and Time Series Visual Builder (TSVB).
+
+## Configuring admin settings
+
+ You can only access, create, or manage alerts for resources for which you have permissions. Access to anomaly detection dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges, and you can manage the settings in **Stack Management**. Access is enabled by default and appears as a feature under **Stack Management** > **Advanced Settings** > **Visualizations**. If the setting is disabled, it does not appear under Stack Management. The setting is disabled at the cluster level through the `opensearch-dashboards.yml` file.
```

Comment:
Didn't 'Stack Management' change to "Dashboards Management'?

Reply:
Yes, good catch. Revised throughout this doc and the alerting doc.